04_numeronym/jacklee/numeronym.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprint(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+
+func numeronyms(vals ...string) []string {
+	r := make([]string, len(vals))
+	for i, v := range vals {
+		var val = []rune(strings.Trim(v, "\n "))
```

Comment:
You can update this to trim the tab character as well by changing it to

```go 
var val = []rune(strings.Trim(v, "\n\t "))
```
or you can also use `strings.TrimSpace()` which trims \n, \t and spaces

```go 
var val = []rune(strings.TrimSpace(v))
```

https://play.golang.org/p/WlMT1CQepia